settings.gradle

```diff
@@ -1 +1,32 @@
+pluginManagement {
+    repositories {
+        google()
+        gradlePluginPortal()
+    }
+}
+
+// https://docs.gradle.com/enterprise/gradle-plugin/
+plugins {
+    id "com.gradle.enterprise" version "3.12.2"
```

Comment:
I don't think we need this plugin

Reply:
f3be6d01a2231b1134f341f8f708c306496628da